code/modules/mob/living/simple_animal/abnormality/zayin/bald.dm

```diff
@@ -60,6 +72,11 @@
 				if(H.z == z)
 					do_bald(H)
 
+/mob/living/simple_animal/hostile/abnormality/bald/BreachEffect(mob/living/carbon/human/user, breach_type)
+	if(breach_type == BREACH_PINK)
+		can_breach = TRUE
```

Comment:
Are you seriously simply editing the variable? It does work, but it's a very weird and roundabout way to handle it.

Reply:
Base abnormality breach code looks for that and it otherwise uses it's normal stats to breach. Other breach methods can also use this to change stats if they like, or this can be adapted for that I suppose, though the argument would be "why not just set it as the default?" which I've done.